.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# Checklist
+=========
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have checked for similar issues and haven't found anything relevant.
+- [ ] This is not a security issue (which should be reported here: https://circleci.com/security/)
+- [ ] I have read [Contribution Guidelines](https://github.com/CircleCI-Public/circleci-cli/blob/master/CONTRIBUTING.md).
+
+## Changes
+=======
+
+- Put itemized changes here, preferably in imperative mood, i.e. "Add
+  `functionA` to `fileB`"
+
+## Rationale
+=========
+
+What was the overarching product goal of this PR as well as any pertinent
+history of changes
+
+## Considerations
+==============
+
+Why you made some of the technical decisions that you made, especially if the
+reasoning is not immediately obvious
+
+## Analytics Events Added
```

Comment:
are analytics events a thing in this repo? Did a [quick search ](https://github.com/CircleCI-Public/circleci-cli/search?q=analytics)on the repo which didn't show anything 🤔

Reply:
no worries. good point, noted for next time 🙂